indi-svbony/svbony_helpers.h

```diff
@@ -0,0 +1,131 @@
+/*
+    ASI CCD Driver
+
+    Copyright (C) 2015 Jasem Mutlaq (mutlaqja@ikarustech.com)
+    Copyright (C) 2018 Leonard Bottleman (leonard@whiteweasel.net)
+    Copyright (C) 2021 Pawel Soja (kernel32.pl@gmail.com)
+
+    This library is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Lesser General Public
+    License as published by the Free Software Foundation; either
+    version 2.1 of the License, or (at your option) any later version.
+
+    This library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public
+    License along with this library; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+#pragma once
+#include <SVBCameraSDK.h>
+#include <indibasetypes.h>
+
+namespace Helpers
+{
+
+const char *toString(SVB_GUIDE_DIRECTION dir)
+{
+    switch (dir)
+    {
+    case SVB_GUIDE_NORTH: return "North";
+    case SVB_GUIDE_SOUTH: return "South";
+    case SVB_GUIDE_EAST:  return "East";
+    case SVB_GUIDE_WEST:  return "West";
+    default:              return "Unknown";
+    }
+}
+
+const char *toString(SVB_BAYER_PATTERN pattern)
+{
+    switch (pattern)
+    {
+    case SVB_BAYER_BG: return "BGGR";
+    case SVB_BAYER_GR: return "GRBG";
+    case SVB_BAYER_GB: return "GBRG";
+    default:           return "RGGB";
```

Comment:
"default" case should be "GRBG".
The Bayer Patter for SVBONY's color camera is "GRBG", so that the correct pattern can be set even in the unlikely event that the Bayer Pattern cannot be obtained.
In that case, "case SVB_BAYER_RG" which is "RGGB" should be added.

In addition, a "note" should be added in the comments so that const char *toString(SVB_BAYER_PATTERN pattern) is not called before the camera is opened.
SVBONY Camera SDK cannot get Bayer patter before camera open.